lib/vdsm/storage/lvm.py

```diff
@@ -1017,49 +1017,58 @@ def getAllVgs(self):
                 self.stats.hit()
         return list(vgs.values())
 
-    def getLv(self, vgName, lvName=None):
+    def getLv(self, vgName, lvName):
         """
-        Get specific LV or all LVs in specified VG.
+        Get specific LV in specified VG.
 
         If there are any stale LVs reload the whole VG, since it would
         cost us around same efforts anyhow and these stale LVs can
         be in the vg.
 
-        We never return Stale or Unreadable LVs when
-        getting all LVs for a VG, but may return a Stale or an Unreadable
-        LV when LV name is specified as argument.
+        May return a Stale or an Unreadable LV.
 
         Arguments:
             vgName (str): VG name to query.
-            lvName (str): Optional LV name.
+            lvName (str): LV name.
 
         Returns:
-            LV nameduple if lvName is specified, otherwise list of LV
-            namedtuple for all lvs in VG vgName.
+            LV nameduple.
         """
+        # vgName, lvName
+        lv = self._lvs.get((vgName, lvName))
+        if not lv or lv.is_stale():
+            self.stats.miss()
+            # while we here reload all the LVs in the VG
+            lvs = self._reloadlvs(vgName)
+            lv = lvs.get((vgName, lvName))
+        else:
+            self.stats.hit()
 
-        if lvName:
-            # vgName, lvName
-            lv = self._lvs.get((vgName, lvName))
-            if not lv or lv.is_stale():
-                self.stats.miss()
-                # while we here reload all the LVs in the VG
-                lvs = self._reloadlvs(vgName)
-                lv = lvs.get((vgName, lvName))
-            else:
-                self.stats.hit()
+        return lv
+
+    def getAllLvs(self, vg_name):
+        """
+        Get all LVs in specified VG.
 
-            return lv
+        If there are any stale LVs reload the whole VG.
 
-        if self._lvs_needs_reload(vgName):
+        Never return Stale or Unreadable LVs.
+
+        Arguments:
+            vgName (str): VG name to query.
+
+        Returns:
+            List of LV namedtuple for all lvs in VG vg_name.
+        """
+        if self._lvs_needs_reload(vg_name):
             self.stats.miss()
-            lvs = self._reloadlvs(vgName)
+            lvs = self._reloadlvs(vg_name)
```

Comment:
Yes, I know. In this case, technically this is a new method (with existing code) and I thought it would be acceptable to change the parameter name.

I'll revert to the previous name and probably add a new commit only for renames (otherwise renaming never happens).